components/user-store/org.wso2.carbon.identity.user.store.configuration/src/main/java/org/wso2/carbon/identity/user/store/configuration/model/UserStoreAttributeMappings.java

```diff
@@ -90,10 +91,12 @@ public void setDefaultUserStoreAttributeMapping(Map<String, UserStoreAttributeDO
     public void setUserStoreAttributeMappings(Map<String, Map<String, ChangedUserStoreAttributeDO>>
                                                       availableUserStoreAttrMappings) {
 
-        for (String userStoreType : availableUserStoreAttrMappings.keySet()) {
-            Map<String, UserStoreAttributeDO> tempMap = getModifiedAttributeMap(availableUserStoreAttrMappings
-                    .get(userStoreType));
-            this.userStoreAttrMappings.put(userStoreType, tempMap);
+        Iterator<Map.Entry<String, Map<String, ChangedUserStoreAttributeDO>>> iterator = availableUserStoreAttrMappings
```

Comment:
These while loop iterators affect the readability of the code. I suggested the following way
```
for (Map.Entry<String, ChangedUserStoreAttributeDO> entry : changedUserStoreAttrMap.entrySet()) {
            if (!clonedAttrMap.containsKey(entry.getKey())) {
                ...
            }
.....
        }
```

Reply:
fixed